syn/Makefile

```diff
@@ -0,0 +1,61 @@
+DEVICE  := hx1k
+PACKAGE := tq144
+CONSTRAINTS_FILE := constraints/remote.pcf
+
+GHDL          ?= ghdl
+GHDL_SYNTH    ?= ghdl
+YOSYS_PLUGIN  ?=
```

Comment:
I have to use `YOSYS_PLUGIN  ?= -m ghdl` here. Do you mind adding it or wouldn't it work with your local build?

Reply:
I can add it. I removed it because some builds of Yosys have ghdl-yosys-plugin built-in. For instance: [open-tool-forge/fpga-toolchain](https://github.com/open-tool-forge/fpga-toolchain). In those cases, passing `-m WHATEVER` to Yosys produces a failure. However, if you local installation is a Linux box, and ghdl-yosys-plugin is built as a module (shared library), then you need it. I will add it. Just provided this explanation for the record.

Reply:
Done.